MatterControlLib/ApplicationView/SettingsValidation.cs

```diff
@@ -43,10 +43,10 @@ namespace MatterHackers.MatterControl
 	public static class SettingsValidation
 	{
 		/// <summary>
-		/// Validates the printer settings satisfy all requirements
+		/// Validates the printer settings satisfy all requirements.
```

Comment:
I've tried to disable this rule (SA1629 - DocumentationTextMustEndWithAPeriod) as it lacks merit IMO. Periods are for structuring English and discrete fields in Xml comments, unlike sentences in paragraphs, don't benefit from the added structure. Hopefully we can sort through the glitch that's preventing disabling this rule and reconsider if it we want to follow its advice


, we don't actually use xml comments and Intellisense summaries don't benefit from